navit/navit_shipped.xml

```diff
@@ -97,6 +97,7 @@ Navigation</text></img>
 				<img src='gui_tools' onclick='network_info()'><text>Network info</text></img>
 			</a>
 			<a name='Route'><text>Route</text>
+				<img src='gui_zoom_route' onclick='zoom_to_route()'><text>Zoom to route</text></img>
```

Comment:
I think that this should be navit.zoom_to_route().
Also, we should probably close the menu after zooming on the route, to display the route.